### PR TITLE
Feature/multi winner election

### PR DESCRIPTION
**Pull request by dasarathg68**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

* **Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)

### CODE REVIEW

File: contract/contracts/Voting/Types.sol [hunk 1]
```diff
@@ -9,13 +9,11 @@ library Types {
     }
 
     struct Candidate {
-        string name;
```
Comment (farrellh1) [inline contract/contracts/Voting/Types.sol:12]: did u delete the name because we can fetch it on db?

File: contract/contracts/Voting/Voting.sol [hunk 2]
```diff
@@ -30,34 +30,45 @@ contract Voting is OwnableUpgradeable, ReentrancyGuardUpgradeable, PausableUpgra
...
+            address[] memory _voters,
+            address[] memory _candidates
+        ) public {
+            require(bytes(_title).length > 0, "Title cannot be empty");
+
+            Types.Proposal storage newProposal = proposalsById[proposalCount];
+            newProposal.id = proposalCount;
+            newProposal.title = _title;
+            newProposal.description = _description;
+            newProposal.draftedBy = _draftedBy;
```
Comment (farrellh1) [inline contract/contracts/Voting/Voting.sol:48]: maybe we can make `drafterBy` is the `msg.sender`

File: contract/contracts/Voting/Types.sol (hunk 1 above)
Comment (dasarathg68) [reply contract/contracts/Voting/Types.sol:12]: Yes!

File: contract/contracts/Voting/Voting.sol (hunk 2 above)
Comment (dasarathg68) [reply contract/contracts/Voting/Voting.sol:48]: Good point

**Pull request merged**